src/Connect/run.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+namespace con {
+
+void run() __attribute__((noreturn));
```

Comment:
Run what? I think this deserves better name, or at least comment.

Reply:
It is in the `namespace con`, so it's `con::run()`. Is that still unclear? (and yes, the `con` should really be eventually renamed to `connect`). What name would you suggest? I'm not happy with java-style whole-sentence names and I don't prefer repeating a namespace or other context inside the name. So, suggestions could be:

* `con::main()`
* `con::main_loop()`

Reply:
Adding comment would be enough for me. I would consider  `main_loop()` easiest to understand, but that's a matter of taste.

Reply:
But all other task main functions are named `..._run`. It's probably better to keep the convention. Adding comment is fine.